DigitalLearningSolutions.Web/Styles/trackingSystem/reports.scss

```diff
@@ -19,8 +19,26 @@ $light-blue: #8EBAFF;
   }
 }
 
+.legend {
+    text-align: center;
+}
+
+.legend li {
+  list-style-type: none;
+  display: inline-block;
+
+  @include mq($until: tablet) {
+    display: list-item;
+  }
+}
+
+.legend-line {
+    stroke-width: 10px;

```

Comment:
The stroke width outside the chart needs to be double to look the same, I haven't been able to work out why.